ros2cli/test/test_ros2cli_daemon.py

```diff
@@ -213,6 +214,8 @@ def test_get_publishers_info_by_topic(daemon_node):
         TEST_TOPIC_PUBLISHER_QOS.durability
     assert test_publisher_info.qos_profile.reliability == \
         TEST_TOPIC_PUBLISHER_QOS.reliability
+    assert test_publisher_info.qos_profile.history == \
+        TEST_TOPIC_PUBLISHER_QOS.history
```

Comment:
same here, i am not sure if this works for all DDS-based rmw implementations. but that we can check with CI.